network/bitcoin_transaction.ksy

```diff
@@ -0,0 +1,147 @@
+meta:
+  id: bitcoin_transaction
+  title: Bitcoin Transaction
+  license: CC-BY-SA-4.0
+  ks-version: 0.1.0
+  endian: le
+doc-ref: |
+  https://bitcoin.org/en/developer-guide#transactions
+  https://en.bitcoin.it/wiki/Transaction
+seq:
+  - id: version
+    type: u4
+    doc: |
+      Version number.
+  - id: nb_of_vins
```

Comment:
I guess the name can be better.

Reply:
Could you please give me the link to the style guide? I didn't find it.

Maybe it's something you should add to the contributing section in the readme file.

Reply:
[The style guide](https://github.com/kaitai-io/kaitai_struct_doc/blob/master/ksy_style_guide.adoc)
[The discussion](https://github.com/kaitai-io/kaitai_struct_doc/pull/7)

Reply:
Thanks!

Reply:
Well, I want to stick as much as possible to the good practices of this project. I just realised that I missed the section in the documentation about the counters:

> For fields that designate number / count of something (in particular, number of repetitions of some other structure), use `num_` prefix and plular form — i.e. `num_questions`, `num_blocks`, `num_nodes`

I guess I should use `num_vins` and `num_vouts`. However, I also read your opinion about it in the discussion you shared with me. Since it's still an open issue, I suggest that I stick to the actual documentation for the moment. What do you think?

Reply:
>What do you think?

I have already said what I think: I dislike hungarian notation because I feel it is inconvenient for me and I guess it may be inconvenient to someone else. But if it is convenient for you, use it. It is your spec, you have created it for yourself, so do what you feel is more convenient for you.

Reply:
I have read the discussion thread so I know your position about it. I wanted to get your opinion on sticking to the actual documentation while waiting to see the outcome of your PR on the style guide. I think it is the right thing to do to keep a global coherence between all the KSY files of the format gallery (which is the purpose of having a style guide).